modules/security/keyvault/modules/secrets.bicep

```diff
@@ -0,0 +1,19 @@
+@description('Specifies the name of the key vault.')
+param keyVaultName string
+
+param secretName string
+
+@secure()
+param secretValue string
+
+resource vault 'Microsoft.KeyVault/vaults@2022-11-01' existing = {
+  name: keyVaultName
+}
+
+resource secret 'Microsoft.KeyVault/vaults/secrets@2022-11-01' = {
+  parent: vault
+  name: secretName
+  properties: {
+    value: secretValue
+  }
+}
```

Comment:
Would it be worth having an output for the secret URI in the event the reference needed to be used downstream in a user's deployment?